src/Kiota.Builder/CodeDOM/CodeConstant.cs

```diff
@@ -95,6 +95,7 @@ public CodeDocumentation Documentation
             Name = $"{codeClass.Name.ToFirstCharacterLowerCase()}{RequestsMetadataSuffix}",
             Kind = CodeConstantKind.RequestsMetadata,
             OriginalCodeElement = codeClass,
+            UriTemplate = $"{codeClass.Name.ToFirstCharacterLowerCase()}{UriTemplateSuffix}",
```

Comment:
why are we using the class name as uri template here?

Reply:
Mentioned it briefly in the PR description but part of the fix in the TypeScript refiner is merging multiple CodeFiles into one CodeFile.
The consequence is that a CodeFile now has multiple UriTemplates in it.
When the TypeScript writer finally gets to writing the request builder's RequestsMetadata constant, it previously only selects the first UriTemplate even if it is the wrong one.
This commit sets UriTemplate in the RequestMetadata CodeConstant so that the TypeScript writer can use it to look up the correct UriTemplate.